js/apps/system/_admin/aardvark/APP/frontend/js/views/graphManagementView.js

```diff
@@ -804,6 +818,17 @@
           );
         }
 
+        if (graph.get('minReplicationFactor')) {
+          tableContent.push(
+            window.modalView.createReadOnlyEntry(
+              'minReplicationFactor',
+              'Minimal Replication factor',
+              graph.get('minReplicationFactor'),
+              'Total number of copies of the data in the cluster.'
```

Comment:
The description is exactly the same as for replicationFactor, maybe there's something more descriptive?